lib/cts/mpx/services.rb

```diff
@@ -32,7 +32,7 @@ def from_url(url)
 
         {
           service:  service.name,
-          endpoint: /data\/([a-zA-Z]*)\//.match(url)[1]
+          endpoint: /data\/([a-zA-Z]*\/Field)\//.match(url)&.[](1) || /data\/([a-zA-Z]*)\//.match(url)&.[](1)
```

Comment:
```suggestion
          endpoint: /data\/([a-zA-Z]*\/Field)\//.match(url)&.[](1) || /data\/([a-zA-Z]*)\//.match(url)&.[](1)
```
I like this, but it'll kill this methods ABC (check the codeclimate details).  

Instead, take the entire line and stuff it inside a private method

```ruby
def endpoint_regex(url)
   .....
end
```

You might even do two of them, one for each side of the `||`